### DISCUSSION TITLE
How do I analyze passwords from argo-secret?

### DISCUSSION THREAD

**Discussion by AllenZMC** [argoproj/argo-cd#9011] [Q&A] [answered]
I have modified admin to log in password. If I forgot how the password I found back?
How do I analyze passwords from argo-secret?
Who can help me, thanks!

**Comment by pasha-codefresh** [comment]
Hey, i think it is answer on your question 
https://codefresh.io/docs/docs/ci-cd-guides/gitops-deployments/

**Reply by AllenZMC** [reply]
Thanks, not it.

**Reply by pasha-codefresh** [reply]
Sorry i put wrong link but Alex already answered

**Comment by alexmt** [comment, marked as answer]
It is impossible to restore password but you can reset it: https://argo-cd.readthedocs.io/en/stable/faq/#i-forgot-the-admin-password-how-do-i-reset-it

**Reply by AllenZMC** [reply]
Thanks
